test/tests/actions/debug_block/tests

```diff
@@ -1,7 +1,14 @@
 [Tests]
-  [./test_debug_block]
+  [./test_show_actions_dependency]
     type = RunApp
     input = 'debug_print_actions_test.i'
-    expect_out = '[DBG][ACT]'
+    expect_out = '[DBG][ACT] Action Dependency Sets'
+    match_literal = true
+  [../]
+  [./test_show_actions_tasks]
+    type = RunApp
+    input = 'debug_print_actions_test.i'
```

Comment:
Oh dang... This test actually writes output (which it probably shouldn't) so you've created a race condition on the (unused) output. I think it's worth fixing now just in case somebody adds more tests here.

Reply:
No actually. See my changes in the input file. I remove the outputs completely and verified there is no output file generated.

Reply:
OK Thanks